chainerx_cc/chainerx/native/native_device/misc.cc

```diff
@@ -53,6 +53,46 @@ class NativeSqrtKernel : public SqrtKernel {
 
 CHAINERX_NATIVE_REGISTER_KERNEL(SqrtKernel, NativeSqrtKernel);
 
+CHAINERX_NATIVE_REGISTER_ELTWISE_DTYPE_BINARY_KERNEL(PowerKernel, { out = chainerx::Power(x1, x2); }, VisitNumericDtype);
```

Comment:
This device implementation might be better to be added in `arithmetic.cc`

Reply:
Agree, fixed. We need to fix these files really.